content/md/supplier-onboarder/manage-products/update-products.md

```diff
@@ -23,10 +23,11 @@ The product form contains 4 parts:
 The header panel shows the following product information:
 * The product name
 * An image of the product (if available)
-* Completeness
-* Breadcrumb to navigate within the [product model and variants](https://help.akeneo.com/pim/serenity/articles/what-about-products-variants.html)
-* Number of missing attributes
-* Filters to access directly to attribute groups or missing attributes
+* The completeness
+* A menu to select the locale to complete
```

Comment:
Do you describe somewhere in the doc what is a locale for Akeneo?

Reply:
Same question for the completeness? (Link to helpcenter?)